okio/src/commonMain/kotlin/okio/-Platform.kt

```diff
@@ -47,3 +47,5 @@ expect interface Closeable {
   @Throws(IOException::class)
   fun close()
 }
+
+expect annotation class JvmDefault()
```

Comment:
Not clear what this does, also should it be `internal`?

Reply:
Yes, definitely

Reply:
Ugh, that’s the right fix but we can’t have Okio’s `@JvmDefault` be internal because it’s typealiased to a public type.

Reply:
Moved to `okio.internal` which is less satisfying but should do the trick.

Reply:
That makes sense. Ideally it should just be present in `stdlib-common`.